.eslintrc

```diff
@@ -29,6 +29,19 @@
         "strict": "error",
     },
     "overrides": [
+        {
+            "files": ["*.mjs", "test/import/package_type/*.js"],
+            "parserOptions": {
+                "ecmaVersion": 2020,
+                "sourceType": "module",
+            },
+        },
+        {
+            "files": ["bin/import-or-require.js"],
+            "parserOptions": {
+                "ecmaVersion": 2020,
+            },
```

Comment:
```suggestion
            "extends": "@ljharb/eslint-config/node",
```

Reply:
:thinking: not sure if i'm missing something here, but the `node` directory in that repo only contains `latest.json` and no `index.js`... so eslintrc complains that the config cannot be found...

Reply:
oops, right
```suggestion
            "extends": "@ljharb/eslint-config/node/latest",
```